apps/aecore/src/aec_peers_pool.erl

```diff
@@ -773,8 +782,8 @@ reference_count(St, PeerId) ->
 find_peer(St, PeerId) ->
     #?ST{peers = Peers} = St,
     case maps:find(PeerId, Peers) of
```

Comment:
Maybe `maps:get/3` would be a better fit here. Just a note, not a blocker

Reply:
That's true all over the module. The structure and choice of calls is pretty bizarre. If you check where `find_peer/2` is called and the way it is called, it is a wrapper around `maps:find/2` that reduces it to the *less* succinct `lists:keyfind/3`. I want to kill all of this weirdness in a cleanup commit later, but there are a *lot* of stupid things like this and I don't have time right now to care about it. In three weeks I will come back and chop all of this nonsense up.